src/commands/revenue.js

```diff
@@ -35,6 +37,13 @@ function definePeriods(args, options) {
     periods.push([start, end]);
   }
 
+  function createYearPeriod(year) {
+    const start = moment(year, 'YYYY').utc();
```

Comment:
This raises a larger issue. I think a bug may have been reintroduced during the rewrite. In this case (and I think also in the case of months) it should not get the time in UTC, but rather the local time for the beginning and end of year/month.